packages/muelu/src/Graph/MatrixTransformation/MueLu_AmalgamationInfo_def.hpp

```diff
@@ -300,12 +271,6 @@ namespace MueLu {
     return ComputeUnamalgamatedImportDofMap(aggregates.GetMap());
   }
 
-  template <class LocalOrdinal, class GlobalOrdinal, class Node>
-  RCP<Xpetra::Map<LocalOrdinal, GlobalOrdinal, Node> > AmalgamationInfo<LocalOrdinal, GlobalOrdinal, Node>::
-  ComputeUnamalgamatedImportDofMap(const Aggregates_kokkos& aggregates) const {
-    return ComputeUnamalgamatedImportDofMap(aggregates.GetMap());
-  }
-
```

Comment:
If we no longer need the two code paths, please remove the trivial methods.

Reply:
This is the only public interface, and it's used in factories like TentativePFactory. Should I make the other interface public?

Reply:
No, just rename.

Reply:
Can you say more? I'm confused about the suggestion.

Reply:
`AmalgamationInfo` had methods with signatures
```
void UnamalgamateAggregates(const Aggregates& aggregates, Teuchos::ArrayRCP<LocalOrdinal>& aggStart, Teuchos::ArrayRCP<GlobalOrdinal>& aggToRowMap) const;
void UnamalgamateAggregatesLO(const Aggregates& aggregates, Teuchos::ArrayRCP<LocalOrdinal>& aggStart, Teuchos::ArrayRCP<LO>& aggToRowMap) const;
void UnamalgamateAggregates(const Aggregates_kokkos& aggregates, Teuchos::ArrayRCP<LocalOrdinal>& aggStart, Teuchos::ArrayRCP<GlobalOrdinal>& aggToRowMap) const;
void UnamalgamateAggregatesLO(const Aggregates_kokkos& aggregates, Teuchos::ArrayRCP<LocalOrdinal>& aggStart, Teuchos::ArrayRCP<LO>& aggToRowMap) const;
void UnamalgamateAggregates(const Teuchos::RCP<const Map> &nodeMap, const RCP<LOVector> &procWinnerVec, const RCP<LOMultiVector> &vertex2AggIdVec, const GO numAggregates, Teuchos::ArrayRCP<LocalOrdinal>& aggStart,Teuchos::ArrayRCP<GlobalOrdinal>& aggToRowMap) const;
void UnamalgamateAggregatesLO(const Teuchos::RCP<const Map> &nodeMap, const RCP<LOVector> &procWinnerVec, const RCP<LOMultiVector> &vertex2AggIdVec, const GO numAggregates, Teuchos::ArrayRCP<LocalOrdinal>& aggStart, Teuchos::ArrayRCP<LO>& aggToRowMap) const
```
All that's needed now is:
```
void UnamalgamateAggregates(const Aggregates& aggregates, Teuchos::ArrayRCP<LocalOrdinal>& aggStart, Teuchos::ArrayRCP<GlobalOrdinal>& aggToRowMap) const;
void UnamalgamateAggregatesLO(const Aggregates& aggregates, Teuchos::ArrayRCP<LocalOrdinal>& aggStart, Teuchos::ArrayRCP<LO>& aggToRowMap) const;
```